File: Patterns/3. Right Triangle Patterns (Top-Left)/24. Right Triangle TL Decreasing Numbers/tldn.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){        // Also possible
        for (int j=n; j>n-i; j--){  // for(j=1;j<=i;j++)
                cout<<j<<" ";       //   cout<<n-j+1
        }
        cout << '\n';
    }
}

/* Output

5 
5 4 
5 4 3 
5 4 3 2 
5 4 3 2 1  

*/ 

File: Patterns/3. Right Triangle Patterns (Top-Left)/24. Right Triangle TL Decreasing Numbers/tldn.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):         # Also Possible
    for j in range(n,n-i,-1):  # for(1,i+1)
        print(j, end = " ")    # print(n-j+1)
    print()

'''
Output:

5 
5 4 
5 4 3 
5 4 3 2 
5 4 3 2 1
    
'''